logistics/scripts/modules/database/database-lifecycle.psm1

```diff
@@ -176,6 +178,7 @@ function Initialize-EdFiDatabaseWithDbDeploy {
             ConnectionString = $csb
             FilePaths        = $filePaths
             Features         = $subTypeNames
+            DatabaseTimeoutInSeconds = $databaseTimeoutInSeconds
```

Comment:
should this variable also work for SQLServer?

Reply:
yes it will work

Reply:
updated